src/js/media-container.js

```diff
@@ -278,6 +277,25 @@ class MediaContainer extends window.HTMLElement {
     }
   }
 
+  set autohide(seconds) {
+    seconds = Number(seconds);
+    this.#autohide = isNaN(seconds) ? 0 : seconds;
+  }
+
+  get autohide() {
+    return this.#autohide === undefined ? 2 : this.#autohide;
+  }
+
+  set breakpoints(val) {
+    if (val === this.breakpoints) return;
+
+    this.setAttribute('breakpoints', val);
+  }
+
+  get breakpoints() {
+    return this.getAttribute('breakpoints') ?? 'sm:384 md:576 lg:768 xl:960';
```

Comment:
I'm not sure if we need this property, but if we have it I'd expect something more helpful than the unsplit string. Probably an object. The setter could still receive either a string or an object.

Reply:
yes, maybe I was a bit quick to add this property.
generally you'd like to reflect props / attrs but this will add unneeded complexity. I will remove it.

Reply:
I'm on the same page and I don't have anything against adding it, it's just up to you if it's worth the extra work right now.